openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -664,3 +664,16 @@ spec:
       "trait:CUSTOM_NUMASIZE_C48_M1459": "required"
       "hw:cpu_cores": "2"
       "vmware:hw_version": "vmx-15"
+  - name: "hana_c288_m4377"
+    id: "311"
+    vcpus: 288
+    ram: 4481528
+    disk: 64
+    extra_specs:
+      "vmware:hv_enabled": "True"
+      "hw_video:ram_max_mb": "16"
+      "resources:CUSTOM_MEMORY_RESERVABLE_MB": "4481528"
+      "resources:CUSTOM_BIGVM": "2"
+      "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+      "hw:cpu_cores": "2"
```

Comment:
This needs to be 3, I'd guess, because we're above 2 * 128 with the `vcpus` set to `288`.